nautobot_ssot/templatetags/humanize_size.py

```diff
@@ -0,0 +1,23 @@
+"""Simple template tag to render a shorter representation of a timedelta object."""
+
+from django import template
+
+
+register = template.Library()
+
+
+@register.filter()
+def humanize_size(size):
+    """Humanize memory size given in bytes.
+
+    Examples:
+        1 => "1.00 B"
```

Comment:
Would be nice if this could be just `1 B` since we don't really deal in fractions of bytes. Not critical though.